app/move/steps/create.js

```diff
@@ -16,32 +15,48 @@ module.exports = {
     resetJourney: true,
     skip: true,
     checkJourney: false,
-    next: 'pnc-search',
+    next: [
+      {
+        fn: req => {
+          return req.session.currentLocation.location_type === 'prison'
+        },
+        next: 'nomis-search',
+      },
+      'pnc-search',
+    ],
+  },
+  '/nomis-search': {
+    entryPoint: true,
+    template: 'move/views/create/nomis-search',
+    pageTitle: 'moves::steps.nomis_offender_no_search_term.heading',
+    buttonText: 'actions::continue',
+    next: 'person-search-results',
+    fields: ['nomis_offender_no_search_term'],
   },
   '/pnc-search': {
-    checkJourney: false,
-    backLink: false,
-    controller: Base,
-    method: 'get',
-    action: '/move/new/pnc-search-results',
```

Comment:
Moving to the next step by doing a get on the next controller/ putting things on the query string seems wrong. Means you can't use the validation features.